site/en/docs/extensions/mv3/webusb/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: "layouts/doc-post.njk"
+title: "WebUSB on ChromeOS"
+seoTitle: "Chrome Extensions: on ChromeOS"
+date: 2023-11-01
+description: The WebUSB API, which exposes non-standard Universal Serial Bus (USB) compatible devices to the web, is available in extensions.
+---
+
+The WebUSB API exposes non-standard Universal Serial Bus (USB) compatible devices to the web. This page describes aspects of the API that are particular to extensions. Refer to MDN for complete details of the [WebUSB](https://developer.mozilla.org/docs/Web/API/WebUSB_API).
+
+## Availability in extensions
+
+ChromeOS 117 or later.
+
+## Permissions
+
+No manifest file permissions are required; however WebUSB triggers the browser's user permission flow.
```

Comment:
```suggestion
No manifest file permissions are required; however to use WebUSB, the [`requestDevice()`](https://developer.mozilla.org/docs/Web/API/USB/requestDevice) method needs to be called from "regular" pages (any of popup, background, tabs) to request access to a device, for which a user gesture is required. Only then the extension service worker can use WebUSB to receive device events.
```

Reply:
I just shared a doc with text in that I had already written and shared with François. Please take a look.